internal/reader/ccip.go

```diff
@@ -99,6 +102,10 @@ func (r *CCIPChainReader) CommitReportsGTETimestamp(
 		return nil, err
 	}
 
+	// ---------------------------------------------------
+	// The following types are used to decode the events
+	// but should be replaced by chain-reader modifiers and use the base cciptypes.CommitReport type.
```

Comment:
Can you create a ticket for this comment and tag me

Reply:
CCIP-2896